LayoutTests/http/tests/scroll-to-text-fragment/start-text-emoji.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><!-- webkit-test-runner [ ScrollToTextFragmentIndicatorEnabled=false ] -->
+<!DOCTYPE html><!-- webkit-test-runner [ ScrollToTextFragmentIndicatorEnabled=true ] -->
```

Comment:
Why are you setting this flag to true? I thought the point was to disable the flash in more places, not enable it sometimes?